tests/cases/matrices_uniform_draw.amber

```diff
@@ -895,13 +895,13 @@ CLEAR gfz_pipeline
 RUN gfz_pipeline DRAW_RECT POS 0 0 SIZE 256 256
 
 EXPECT framebuffer IDX 0     0 SIZE 85 85 EQ_RGBA   0   0   0 255
-EXPECT framebuffer IDX 85    0 SIZE 85 85 EQ_RGBA  96  96  96 255
-EXPECT framebuffer IDX 171   0 SIZE 85 85 EQ_RGBA 127 127 127 255
+EXPECT framebuffer IDX 85    0 SIZE 85 85 EQ_RGBA  96  96  96 255 TOLERANCE 1 1 1 0
```

Comment:
Why add all the tolerance values?

Reply:
These were failing on Intel/Mesa because of results being off by one. Unless we can be sure the results need to be exact I'm suggesting to use tolerance now that it's supported.

Reply:
Ok, fair enough. I'd like to make sure we have a couple of these that don't take TOLERANCE just so we can make sure they keep working, but if we have to convert some that seems OK.